clients/hydrophone/hydrophone_test.go

```diff
@@ -80,6 +83,74 @@ func TestGetPendingInvitations(t *testing.T) {
 	testError(t, hydrophoneClient, testToken, "invite")
 }
 
+func TestGetSentInvitations(t *testing.T) {
```

Comment:
running with coverage show that it misses the errors cases

Reply:
There is an error case that I don't know how to reach... If you can show me that would be great :slightly_smiling_face: